src/chocolatey.tests.integration/context/upgradepackage/1.1.1-beta.1/tools/graphical.exe.gui

```diff
@@ -0,0 +1 @@
+1.1.0
```

Comment:
Not completely sure of this one, I don't mind removing it. It was merely a copy from our existing packages that already had this in the file.

Reply:
Instead of removing this, I have replaced the value with the actual version of the package.
As well as updated other places I had missed that should have been updated as well.